openmdao/core/tests/test_check_derivs.py

```diff
@@ -6,6 +6,7 @@
 import unittest
 
 import numpy as np
+import sys
```

Comment:
has the code that needed this import been deleted?

Reply:
fixed